src/components/Tree/Tree.tsx

```diff
@@ -69,9 +69,7 @@ class Tree extends UIComponent<TreeProps, any> {
     return _.map(items, obj => {
       return TreeListItem.create(obj.title, {
         defaultProps: {
-          styles: styles.listItem,
```

Comment:
We will need these variables for theming.

Reply:
Can you please provide an example?

https://github.com/stardust-ui/react/blob/master/src/components/Header/Header.tsx#L85
I'm almost sure that we don't need to pass down all variables by tree.

Reply:
I was using the pattern used here in button https://github.com/stardust-ui/react/blob/28c8307fa24dc8e5adbdb882f1c0c2d8cfbeeec2/src/components/Button/Button.tsx#L171